mkt/lookup/tests/test_views.py

```diff
@@ -788,11 +788,21 @@ def test_all_results(self):
 
     def test_statuses(self):
         for status, api_status in mkt.STATUS_CHOICES_API_v2.items():
+            if status == mkt.STATUS_DELETED:
+                # Deleted status is too special to recover from, so it needs
```

Comment:
It's isolated, because you can't recover from an app in `status=STATUS_DELETED` anymore. So the loop would fail when changing from deleted to another status.

Ideally this loop should not be made inside the test at all to avoid similar problems with other statuses.